example/index.ts

```diff
@@ -0,0 +1,24 @@
+import LuckyBot from '../src';
+
+const ninetyMin = 90 * 60 * 1000;
+const run = async () => {
+  const user = process.env.LUCKYBOT_USER;
+  const pass = process.env.LUCKYBOT_PASS;
+  const hastag = process.env.LUCKYBOT_HASTAG || 'nepaltravel';
+
+  if (!user || !pass) {
+    throw new Error('No username or password')
+  }
+  
+  const bot = new LuckyBot(user, pass, {debug: true});
+
+  console.log('LOGIN...')
+  await bot.login();
+  console.log('LIKING PHOTOS...')
+  await bot.likePhotos(hastag, {maxLikes: 5});
+  console.log('CLOSE...')
+  await bot.close();
+};
+
+run();
+setInterval(run, ninetyMin)
```

Comment:
@MP0w I think we won't need cronjob at all. We can just keep the process running forever with a `setInterval`.

Potentially we can even offer that functionality in the library if we want to make it easier for the users

Reply:
Yeah I guess other libs are just sleeping and going, sleeping and going